### ISSUE TITLE
Charter not rendering

### ISSUE THREAD

**Issue by piktonus97m** [fabiocolacio/Marker#319]
Hello team, 

Im currently using the latest version of PopOs as well as the latest version of Marker, and the current rendering for Charter is not showing up the chart. I'm attaching the screen. 

![image](https://user-images.githubusercontent.com/37163970/87861564-a8f67e00-c904-11ea-90db-b641ae0dc3b7.png)

**Comment by Mandarancio** [comment]
Hi @piktonus97m ,
I tried to reproduce the bug with the current master code and I did not encounter any problem:
![marker_charter](https://user-images.githubusercontent.com/276686/87921566-090f3080-ca7b-11ea-8b70-2ca31b11d2b6.png)
I suppose you have enabled charter in the options as it shows in this screenshot:
![marker_options](https://user-images.githubusercontent.com/276686/87921567-09a7c700-ca7b-11ea-855f-b662c2aed9f3.png)

How did you install Marker?

Cheers,
M.

**Comment by piktonus97m** [comment]
Hello, @Mandarancio yes I have enable the charter option on the preferences. I just tried on my Work-Computer; a RHEL 8.2. Also is not working on both machines I installed using Flatpak/Flathub. 
Thanks in advance.

**Comment by Mandarancio** [comment]
Hi @piktonus97m, I am currently using Fedora, so it should not be related with RHEL.
More probably the problem is  related with the packaging, could you try to install it manually (following the instruction on the Readme) and see if the problem is still there?

**Comment by vitkrn** [comment]
Hello everyone. Charter is not working for me too. Using Fedora 33.

**Comment by myprs** [comment]
Hi Team,

same problem here on Ubuntu 20.04.4 LTS, 

``` console
$ flatpak list 
Name                                                 Application ID                                                   Version                 Zweig                   Installation
[..]
Marker                                               com.github.fabiocolacio.marker                                   2020.04.04              stable                  system
[..]
```

**Comment by myprs** [comment]
Uninstalling the flatpak and installing the  ppa: solved the problem here, so this seems to be a problem in the flatpak.

> Hi Team,
> 
> same problem here on Ubuntu 20.04.4 LTS,
>

**Comment by apandada1** [comment]
I cannot reproduce it in the PPA package in Kubuntu 22.04.

**Comment by johnblommers** [comment]
Agreed that it's an issue with the Flatpak edition of Marker at least on PopOS.

My solution was to remove it and install instead using `snap install marker`, remembering to enable the feature of course ;-)

**Comment by fcosanabria** [comment]
The issue seems to be related to Flatpak version of Marker. Closing the issue.
